SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -76,6 +77,8 @@
   @Min(value = 1, message = "Must be positive and non-zero")
   private int defaultBounceExpirationMinutes = 60;
 
+  private String defaultDatePattern = "MMM dd h:mm:ss a zzz";
```

Comment:
I think we should still make it more specific. At the moment, it is only used for mail, so let's make the name reflect that. Also it isn't really a default that we are falling back on, it is just a setting that will be used by everything. So maybe just calling it `mailerDatePattern` ?